codec-http2/src/test/java/io/netty/handler/codec/http2/HpackDecoderTest.java

```diff
@@ -512,4 +512,91 @@ public void testAccountForHeaderOverhead() throws Exception {
             in.release();
         }
     }
+
+    @Test
+    public void unknownPseudoHeader() throws Exception {
+        ByteBuf in = Unpooled.buffer(200);
+        try {
+            HpackEncoder hpackEncoder = new HpackEncoder(true);
+
+            Http2Headers toEncode = new DefaultHttp2Headers();
+            toEncode.add(":test", "1");
+            hpackEncoder.encodeHeaders(1, in, toEncode, NEVER_SENSITIVE);
+
+            // decode the headers, we should get an exception, but
+            // the decoded headers object should contain all of the headers
```

Comment:
this makes it sound like you would expect `decoded` to contain `:test=1` after the decode operation, but that is not the case. Can you clarify this comment?

Reply:
- [X] Remove comment